packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_inputs.dart

```diff
@@ -270,32 +270,11 @@ mixin _PropertyInputMixin<T extends StatefulWidget, U> on State<T> {
     });
   }
 
-  void _maybeHandleServerError(
-    EditArgumentResponse? errorResponse, {
-    required EditableProperty property,
-  }) {
+  void _maybeHandleServerError(EditArgumentResponse? errorResponse) {
     if (errorResponse == null || errorResponse.success) return;
     setState(() {
-      _serverError = _errorMessage(errorResponse.errorType, property: property);
+      _serverError =
+          errorResponse.errorType?.message ?? 'Encountered unknown error.';
```

Comment:
do we still want to include the property in this error message? I think it would be helpful:
`'${errorResponse.errorType?.message ?? 'Encountered unknown error.'} Property: $property.'`

Reply:
Done!